### PR TITLE
Feature errored vs failed status distinction

### PR DESCRIPTION
**Pull request by flbulgarelli**
Related to https://github.com/mumuki/mumuki-laboratory/issues/631
Fixes https://github.com/mumuki/mumuki-laboratory/issues/801
Fixes https://github.com/mumuki/mumuki-laboratory/issues/800

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -8,3 +8,19 @@
 @import "application/errors";
 @import "application/alerts";
 
+$brand-broken: #651621;
```
Comment (flbulgarelli) [inline app/assets/stylesheets/application.scss]: This should go into mumuki-styles, right?

**Comment by flbulgarelli** [comment]
## Failed assignment


screencapture-localmumuki-io-3000-central-exercises-607-programacion-funcional-valores-y-funciones-mas-valores-1502689197710

**Comment by flbulgarelli** [comment]
@faloi @LauraMangifesta @asanzo @fedescarpa @aguspina

**Comment by fedescarpa** [comment]
LGTM

**Comment by fedescarpa** [comment]
Rebase to master and merge it

**Comment by flbulgarelli** [comment]
> However, I propose a minor change: the failed and errored messages are very similar. I'd remove the "ups" from the failed message, to emphatize the other.

:+1:

**Pull request merged**